tool/clean/clean_eks/clean_eks.go

```diff
@@ -6,16 +6,18 @@ package main
 import (
 	"context"
 	"log"
-	"strings"
 	"time"
 
 	"github.com/aws/aws-sdk-go-v2/aws"
 	"github.com/aws/aws-sdk-go-v2/config"
 	"github.com/aws/aws-sdk-go-v2/service/eks"
+	"github.com/grafana/regexp"
```

Comment:
Is there something special about the grafana regexp library?

Why not use the standard (or w/e we use in CWAgent)?